.boatsrc

```diff
@@ -1,14 +1,17 @@
 {
   "nunjucksOptions": {
-    "tags": {
-    }
-  },
-  "jsonSchemaRefParserBundleOpts": {
+    "tags": {}
   },
+  "jsonSchemaRefParserBundleOpts": {},
   "permissionConfig": {
     "globalPrefix": false
   },
   "picomatchOptions": {
     "bash": true
+  },
+  "paths": {
+    "@oa3Mixins": "srcOA3/mixins/",
```

Comment:
Unlike typescript, this is a single value, not an array. This seemed a reasonable compromise compared to searching all the relevant paths to find a file.